python_backend/api/v1/schemas/quality.py

```
@@ -1,31 +1,223 @@
+    confidence_scores: Dict[str, float] = Field(default={}, description="신뢰도 점수들")
+    fallback_reason: Optional[str] = Field(None, description="fallback 사용 이유")
+
+class DetailedCompanyQualityResponse(CompanyQualityAnalysisResponse):
+    """상세 분석 정보 포함 응답 (detailed=true일 때)"""
+    debugInfo: Optional[AnalysisDebugInfo] = Field(None, description="디버그 정보")
+    enterpriseAnalysis: Optional[Dict[str, Any]] = Field(None, description="기업 상세 분석")
+    usageRecommendations: Optional[Dict[str, List[str]]] = Field(None, description="사용 권장사항")
+
+
+# 에러 응답 스키마
+class CompanyErrorResponse(BaseModel):
```

Ruff (0.13.1)

Redefinition of unused `CompanyErrorResponse` from line 194

(F811)